CLAUDE.md

```
@@ -17,3 +17,7 @@ Index for coding assistants (Claude Code, Copilot, etc.). The detailed guidance
 - Testing: [docs/development/TESTING.md](docs/development/TESTING.md)
 - Architecture: [src/ARCHITECTURE.md](src/ARCHITECTURE.md)
 - Configuration: [src/docs/CONFIGURATION.md](src/docs/CONFIGURATION.md)
+
+
+## Secrets and API Keys
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)